vsutil.py

```diff
@@ -166,3 +166,10 @@ def is_image(filename: str) -> bool:
     Returns true if a filename refers to an image.
     """
     return mimetypes.types_map.get(os.path.splitext(filename)[-1], "").startswith("image/")
+
+
+def is_luma_only(clip: vs.VideoNode) -> Bool:
+    """
+    Returns true if a clip has only one plane
+    """
+    return format.num_planes == 1
```

Comment:
Did you mean `clip.format.num_planes`?

Either way, this is a function for simple comparison I don't personally feel worth adding a function for. Besides, you only actually test for the number of planes, not whether the plane is luma, so writing the actual expression in code is more explicit and easier to understand in a context.

Reply:
Yeah, fixed that.

I think it is general enough to be worthwhile to have in vsutil, but I do think the name needs to be changed. That's why I asked for name suggestions in the original post.